webfront/views/cache.py

```diff
@@ -18,6 +18,29 @@
 
 names = [ep["name"].lower() for ep in endpoints]
 
+short_life_parameters = [
+    "cursor",
+    "size",
+    "go_terms",
+    "ida_ignore",
+    "ida_search",
+    "format",
+    "page_size",
+    "search",
+]
+
+no_cache_modifiers = [
+    "extra_features",
+    "residues",
+    "isoforms",
+    "ida",
+    "taxa",
+    "model:"
+    "annotation:info",
+    "subfamilies",
+    "subfamily",
+    "page_size",
```

Comment:
What if `page_size` is passed with the default page size (20 items per page)? Shouldn't it be cached?

Reply:
Yeah good point. I was thinking that that case will be solved by the posterior use of `canonical()` in the cache itself. But now that you make me think about it, it will get rejected before it gets there. So I'm using `canonical` when checking the lifespan